com/course/zenkouv/src/test/java/calculator/CalculatorTest.java

```diff
@@ -0,0 +1,63 @@
+package calculator;
+
+import lecture5.util.operation.Operation;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+public class CalculatorTest {
+
+    private final Operation operation = new Operation();
```

Comment:
create new instance of `Operation ` in test and make all tests depends on it (just for training )

Reply:
fix